doc/ref_model/chapters/chapter01.md

```diff
@@ -68,5 +39,54 @@ The document starts from the abstract and as it progresses it increasingly gets
 - **Appendix A - Guidelines For Application Vendors**: More details related to how the applications will interface with the cloud infrastructure through APIs (including Cloud Infrastructure Manager and CaaS).
   > **Audience**: This chapter is written for architects, developers and others who need to deploy infrastructure or develop applications.
 
+<a name="1.2"></a>
+## 1.2 Scope
+This document focuses on the documenting the higher level concepts that are needed to identify **Reference Model**. **Figure 1-1** below highlights its scope in more details.
+
+<p align="center"><img src="../figures/ch01_scope.png" alt="scope" title="Scope" width="100%"/></p>
+<p align="center"><b>Figure 1-1:</b> Scope of Reference Model</p>
+
+This document specifies:
+- **Cloud Infrastructure abstraction**: in context with how it interacts with the other components required to build a complete system that supports workloads deployed in Virtual Machines (VNF) or containers (CNF).
+  - **Cloud Infrastructure metrics & capabilities**: A set of metrics and capabilities for the cloud infrastructure which workloads require to perform telco scale network functions.
+  - **Infrastructure profiles catalogue**: A catalogue of standard profiles needed in order to abstract the infrastructure from workloads. With a limited and well-defined set of profiles with well understood characteristics, workload compatibility and performance predictability can be achieved.
+
+- Cloud Infrastructure Software and Hardware profiling
+  - **Cloud Infrastructure software profiles and configurations**: These are software profiles and configurations that map directly to the infrastructure profiles within the infrastructure profiles catalogue.
+  - **Cloud Infrastructure hardware profiles and configurations**: These are hardware profiles and configurations which are suitable for the defined cloud infrastructure software profiles & configurations.
+
+- Conformance and verification
+  - **Conformance programs**: This defines the requirements for certification and validation programs for both workloads and cloud infrastructure.
```

Comment:
since programs plural; also certification is loaded term; primary focus cloud infrastructure and also workload conformance is highly limited in scope
```suggestion
  - **Conformance programs**: These define the requirements for verification and validation programs for both the cloud infrastructure and workloads.
```